emcmake.py

```diff
@@ -34,11 +34,10 @@ def has_substr(args, substr):
   # Append the Emscripten toolchain file if the user didn't specify one.
   if not has_substr(args, '-DCMAKE_TOOLCHAIN_FILE'):
     args.append('-DCMAKE_TOOLCHAIN_FILE=' + utils.path_from_root('cmake', 'Modules', 'Platform', 'Emscripten.cmake'))
-  node_js = config.NODE_JS
 
   if not has_substr(args, '-DCMAKE_CROSSCOMPILING_EMULATOR'):
     node_js = config.NODE_JS[0].replace('"', '\"')
-    args.append('-DCMAKE_CROSSCOMPILING_EMULATOR="%s"' % node_js)
+    args.append(f'-DCMAKE_CROSSCOMPILING_EMULATOR={node_js}')
```

Comment:
Yes the fix is to remove the quote marks which were causing issues with CMake, especially on newer version.

Because args is being directly to subprocess.call there is no need to quote spaces.. cmake will see this is one big single argument.. there is no shell parsing this string before it goes to cmake.   I will verify locally with a space in my node path just to be sure..

Reply:
Verified that spaces in node path work fine after this change.

Reply:
I see, thanks.